docs/user_guide/scripts/02_medrecord_intro.py

```diff
@@ -29,13 +29,8 @@
 
 record = mm.MedRecord.builder().add_nodes((patients, "ID"), group="Patients").build()
 
-# TODO: Adding new nodes to group must be added as issue in next MedModels weekly
-record.add_nodes((medications, "ID"))
+record.add_nodes((medications, "ID"),group="Medications")
```

Comment:
small comment on spacing style
```suggestion
record.add_nodes((medications, "ID"), group="Medications")
```